doc/book/admin/instance_config.rst

```diff
@@ -1,177 +1,204 @@
 .. _admin-instance_config:
+.. _admin-instance-environment-overview:
+.. _admin-tt_config_file:
 
-Instance configuration
-======================
+Application environment
+=======================
 
-For each Tarantool instance, you need two files:
+This section provides a high-level overview on how to prepare a Tarantool application for deployment
+and how the application's environment and layout might look.
+This information is helpful for understanding how to administer Tarantool instances using :ref:`tt CLI <tt-cli>` in both development and production environments.
 
-* [Optional] An :ref:`application file <app_server-launching_app>` with
-  instance-specific logic. Put this file into the ``/usr/share/tarantool/``
-  directory.
+The main steps of creating and preparing the application for deployment are:
 
-  For example, ``/usr/share/tarantool/my_app.lua`` (here we implement it as a
-  :ref:`Lua module <app_server-modules>` that bootstraps the database and
-  exports ``start()`` function for API calls):
+1.  :ref:`Initializing a local environment <admin-instance_config-init-environment>`.
```

Comment:
Not sure about "local" here. Maybe just "a tt environment"?
Local env is typical for dev time, but on real clusters the system tt mode can be used.

Reply:
BTW, if you use the exact section name in a link, you don't need to write it:
```
:ref:`instance_config-init-environment`
```

Reply:
> Not sure about "local" here. Maybe just "a tt environment"?
Local env is typical for dev time, but on real clusters the system tt mode can be used.

This section emphasise that the environment is created for development. So, I'd keep `local` or maybe changed to `development`.